### PR TITLE
feat:  push to old fleets on nebra os cloud #25

### PR DESCRIPTION
**Pull request by pritamghanghas**
* added deployemnt for old fleets.

**[Issue](https://github.com/NebraLtd/NebraOS/issues/25)**

- Link: https://github.com/NebraLtd/NebraOS/issues/25
- Summary:

**How**

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        if: matrix.sbc == 'pisces'
+        run: |
+          python gen_docker_compose.py pisces -o docker-compose.yml
+          cp balena/balena.yml.pisces balena.yml
+      - name: Balena Deploy (Nebra ROCK Pi)
+        uses: nebraltd/balena-cli-action@v15.0.3
+        if: matrix.sbc == 'rockpi' && success()
+        with:
+          balena_api_token: ${{secrets.NEBRAOS_CLOUD_TOKEN}}
+          balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }}${{ env.ROCKPI }} --debug --nocache --build"
```
Comment (MuratUrsavas) [inline .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml:139]: You need `nebra_cloud: true` for all of the deployment steps

**Review by MuratUrsavas — CHANGES_REQUESTED** [review]

File: .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        with:
+          balena_api_token: ${{secrets.NEBRAOS_CLOUD_TOKEN}}
+          balena_command: "deploy nebraltd/helium-${{ matrix.sbc }} --debug --nocache --build"
+      - name: Trigger build action in helium-${{ matrix.sbc }} repo
+        uses: peter-evans/repository-dispatch@v2
+        if: ( matrix.sbc == 'rak' || matrix.sbc == 'pisces' || matrix.sbc == 'og' || matrix.sbc == 'sensecap' || matrix.sbc == 'finestra' || matrix.sbc == 'controllino' || matrix.sbc == 'cotx' || matrix.sbc == 'pantherx1' || matrix.sbc == 'linxdot' || matrix.sbc == 'linxdot-rkcm3' || matrix.sbc == 'syncrobit' || matrix.sbc == 'syncrobit-rkcm3' || matrix.sbc == 'pycom' || matrix.sbc == 'risinghf' || matrix.sbc == 'midas' ) && success()
+        with:
+          token: ${{ secrets.MR_BUMP }}
+          event-type: build-open-fleet
+          repository: NebraLtd/helium-${{ matrix.sbc }}
```
[the remark this answers is not in the document]
Comment (shawaj) [reply .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml:176]: @MuratUrsavas we will still need different fleets for different manufacturers as they're different device types

[the remark this answers is not in the document]
Comment (pritamghanghas) [reply .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml:176]: yes, we can get rid of those but later. Right now, the way migration will work is "a device in x fleet in balena-cloud will come to a fleet by same name in nebra cloud". We can consolidate when migration is complete.

File: .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml (hunk 1 above)
Comment (pritamghanghas) [reply .github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml:139]: yeah, forgot.

**Review by MuratUrsavas — APPROVED** [review]

**Pull request merged**